gulp-config.js

```diff
@@ -3,10 +3,7 @@
 ((() => {
   const themeDir = './';
   const paths = {
-    js: [
-      `${themeDir}/js/**/*.js`,
```

Comment:
What if you have javascript that isn't tied to a component? Is that a thing you deal with? Do you just put it in a pattern? It may be nice to still have this for "utility" JS that often gets written.

Reply:
Maybe, but it's not in there by default and we encourage everything being component-based. I'm fine with leaving it out and then if a person chooses to add this directory (which doesn't exist currently), then they can use local.gulp-config.js to override.